<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Window</string>
  </property>
  <layout class="QGridLayout" name="win_lay">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data sets:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="Data_sets"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Files to plot :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="CurvesToPlot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="dsq">
             <property name="text">
              <string>dS(q, t)</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="qdsq">
             <property name="text">
              <string>q*dS(q, t)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="qqdsq">
             <property name="text">
              <string>q**2*dS(q, t)</string>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="Errorbars">
           <property name="text">
            <string>Errorbars</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonUpdate">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ResetZoom">
           <property name="text">
            <string>Reset Zoom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>200</horstretch>
         <verstretch>200</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="plot_vl"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
